citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -532,7 +536,9 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
       			out.println("\t\t\t</table>");
        		    out.println("\t\t</div>");
 
-			    out.println("\t\t<div class=\"availabilityHeader\"></div>");
+
+			    out.println("\t\t<div class=\"availabilityHeader\" style=\"color:" +
+					    ServerConfigurationService.getString("official.institution.background.colour") + "\">" + "<strong>Availability</strong>" + "</div>");
```

Comment:
It's already used in WL-3948. I gave it a more name as we might want to use Oxford colours in other places (e.g., the portal header) in Weblearn